content/en/continuous_integration/intelligent_test_runner/javascript.md

```diff
@@ -27,32 +27,83 @@ Intelligent Test Runner is only supported in the following versions and testing
 * `cypress>=6.7.0`
   * From `dd-trace>=4.2.0`, `dd-trace>=3.23.0` or `dd-trace>=2.36.0`.
   * Instrument your web application with code coverage: see more in [Cypress setup](#cypress-setup)
+* If you're reporting data through the Datadog Agent, use v6.40+/v7.40+.
 
 ## Setup
 
+### Test Visibility Setup
+
 Prior to setting up Intelligent Test Runner, set up [Test Visibility for Javascript and Typescript][2]. If you are reporting data through the Agent, use v6.40+/v7.40+.
 
-To enable Intelligent Test Runner, set the following environment variables:
+### UI activation
+You or a user in your organization with "Intelligent Test Runner Activation" permissions must activate the Intelligent Test Runner on the [Test Service Settings][3] page.
+
+{{< img src="continuous_integration/itr_overview.png" alt="Intelligent test runner enabled in test service settings in the CI section of Datadog.">}}
+
+### Configuring the test runner environment
+
+{{< tabs >}}
+
+{{% tab "On-Premises CI Provider (Datadog Agent)" %}}
+
+#### Additional Datadog Agent configuration
+
+In addition to the [Datadog API Key][1], the [Datadog Application key][2] must be specified in the [Agent Configuration File][3]
+
+`app_key` (Required)
+: The [Datadog Application key][2] used to query the tests to be skipped.<br/>
+**Default**: `(empty)`
+
+`api_key` (Required)
+: The [Datadog API key][1] used to upload the test results.<br/>
+**Default**: `(empty)`
+
+After configuring the Datadog Agent, run your tests as you normally do:
+
+{{< code-block lang="shell" >}}
+NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app yarn test
+{{< /code-block >}}
+
+[1]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: https://app.datadoghq.com/organization-settings/application-keys
+[3]: /agent/guide/agent-configuration-files
+{{% /tab %}}
+
+{{% tab "Cloud CI provider (Agentless)" %}}
+
+#### Environment variables
+
+To enable Intelligent Test Runner in Agentless mode, set the following environment variables:
+
+`DD_CIVISIBILITY_AGENTLESS_ENABLED=true` (Required)
```

Comment:
Hm, my docs assume that `DD_CIVISIBILITY_AGENTLESS_ENABLED` and `DD_API_KEY` will already be there because we've already told them, above, to follow the Test Visibility setup.

I think it might become unwieldy to repeat all the keys involved on every page.